REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.4
+julia 0.5
 BinDeps
-Compat 0.8
+Compat 0.26.0
```

Comment:
Fixed in 2c64559d1f266b7f12e4c0531a70994279d9d2d3.  I'll update the METADATA PR in a sec.